Tarantool fails to compile on Ubuntu 18.04

Compiling it with the system's default gcc leads to the following error:

```
[ 87%] Building C object src/box/CMakeFiles/box.dir/sysview.c.o
/tmp/tarantool_install2626529564/src/box/sysview.c:286:29: error: initializer element is not constant
 const uint32_t PRIV_WRDAG = PRIV_WRDA | PRIV_GRANT;
                             ^~~~~~~~~
```

Note the definition of the problematic (non-const) symbol:

```C
const uint32_t PRIV_WRDA = PRIV_W | PRIV_D | PRIV_A | PRIV_R;
```

Guilty commit: https://github.com/tarantool/tarantool/commit/1fd4404bf66eff7f7d072e9883a537922d06f125#diff-0c724a56e43bbc8fdec98ef5a895c07b9516ecc928c5f392d0afd738c5b7f0c5R286

Found by @elhimov.

--- a/src/box/sysview.c
+++ b/src/box/sysview.c
@@ -282,8 +282,10 @@ sysview_space_execute_upsert(struct space *space, struct txn *txn,
  * 7. User is parent for the user/role.
  */
 
-const uint32_t PRIV_WRDA = PRIV_W | PRIV_D | PRIV_A | PRIV_R;
-const uint32_t PRIV_WRDAG = PRIV_WRDA | PRIV_GRANT;
+enum {
+	PRIV_WRDA = PRIV_W | PRIV_D | PRIV_A | PRIV_R,
+	PRIV_WRDAG = PRIV_WRDA | PRIV_GRANT,
+};
 
 static bool
 vspace_filter(struct space *source, struct tuple *tuple)

